scrips/utils.js

```diff
@@ -14,6 +14,47 @@ const createDomElement = (domEl = "div", classToAssign = "", idToAssign = "") =>
     return div;
 };
 
-const formatHours = (hour) => (hour < 10 ? `0${hour}` : `${hour}`);
+const getLongWeekDayName = (shortName) => {
+    const weekDaysNames = [
+        { short: "Mon", long: "Monday" },
+        { short: "Tue", long: "Tuesday" },
+        { short: "Wed", long: "Wednesday" },
+        { short: "Thu", long: "Thursday" },
+        { short: "Fri", long: "Friday" },
+        { short: "Sat", long: "Saturday" },
+        { short: "Sun", long: "Sunday" },
+    ];
 
-export { setSameWidth, createDomElement, formatHours };
+    return weekDaysNames.find((name) => name.short === shortName).long;
+};
+
+const getLongMonthName = (shortName) => {
+    const monthNames = [
```

Comment:
Complexity of this function is `O(n)` you can make it `O(1)`:

```suggestion
const shortMonthNameToLongNameMap = {
  Jan: "January",
  ....define other months
};

const getLongMonthName = (shortMonthName) => shortMonthNameToLongNameMap[shortMonthName]
```

P.S ideally you should not have things like shortMonthName. These should be defined as an enum:

```
const MONTHS = {
 JANUARY = 'january',
 ....
}

const getMonthName = (month) => monthNameMap[month]
```